hazelcast/src/test/java/com/hazelcast/map/impl/tx/MapTransactionTest.java

```diff
@@ -408,14 +413,19 @@ public Boolean execute(TransactionalTaskContext context) throws TransactionExcep
         });
         assertTrue(b);
 
-        IMap map1 = h2.getMap("putWithTTL");
-        assertEquals("value", map1.get("1"));
+        final Object result = map2.get("1");
+        if (deletedCounter.get() > 0) {
+            System.out.println("Value was already removed from map, quitting test");
```

Comment:
This was changed. Also, the atomic counter was removed due to the fact that the interceptor can be serialized and invoked on an another instance, thus failing to pass assertion. As the counter was irrelevant and we already have one assertion that the map contains the value, we just need to assert eventually that the map does not contain the value anymore. WDYT?